src/functions/usdcreateserverinvite.md

```diff
@@ -51,11 +51,7 @@ bot.command({
 bot.command({
   name: "create-server-invite", 
   code: `
-  $createServerInvite[$guildID;{
-    "maxAge": 0,
-    "maxUses": 0,
-    "temporary": false
-  }]
```

Comment:
what i meant was 
make 2 examples for params

one that uses json
and one that uses normal